src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -331,7 +331,11 @@ def is_already_paid(self, order, installment):
         payload = {
             "orderId": str(order.id),
         }
-        response_json = self._call_api(url, payload)
+        try:
+            response_json = self._call_api(url, payload)
+        except exceptions.PaymentProviderAPIException as error:
+            order.set_installment_refused(installment["id"])
+            return False
```

Comment:
I'm afraid this will not work. Furthermore in case of API exception (Timeout for example) we will able again to debit an installment twice.